PeriodogramAnalysis/MultitaperPowerSpectrum.py

```diff
@@ -0,0 +1,86 @@
+import numpy as np
+from scipy.fft import rfft, rfftfreq
+from scipy.signal.windows import dpss
+
+
+def multitaper_psd(
+    x, sfreq, fmin=0.0, fmax=np.inf, bandwidth=None, adaptive=True, low_bias=True
+):
+    """
+    Compute multitaper power spectral density (PSD) of a given signal.
```

Comment:
Which reference are you using for multitapered power spectrum?

Reply:
I added Parts of this code is inspired from MNE: https://mne.tools/stable/index.html in docstring, it used similar methods as MNE.